app/controllers/items_controller.rb

```diff
@@ -547,7 +547,7 @@ def set_rights
   end
   # set the content type in the content metadata
   def set_content_type
-    unless %w(book file image map manuscript).include? params[:new_content_type]
+    unless Constants::CONTENT_TYPES.include? params[:new_content_type]
       render :status => :forbidden, :text => 'Invalid new content type.'
```

Comment:
I created a new PR for it as I found a few cases in items controller: https://github.com/sul-dlss/argo/issues/500